python/import_util.py

```diff
@@ -0,0 +1,93 @@
+from mage.export_import_util.parameters import Parameter
+import mgp
+import json as js
+
+
+def create_vertex(ctx, properties, labels):
+    vertex = ctx.graph.create_vertex()
+    vertex_properties = vertex.properties
+
+    for key, value in properties.items():
+        vertex_properties[key] = value
```

Comment:
Does this mutate the object?

Reply:
I was following the [documentation](https://memgraph.com/docs/memgraph/2.1.1/reference-guide/query-modules/api/python-api#example-usage-1) and did this in the same way. I supposed that it changes the object inside Memgraph, and while testing it, I realized it does. Let me know if I understood your question well.

Reply:
Yes, thanks for clearing that one